src/figuresClasses.ts

```diff
@@ -1,11 +1,88 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
 
-export class Triangle implements Figure {}
+  color: string;
 
-export class Circle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  constructor(
+    color: string,
+    public a: number = 0,
+    public b: number = 0,
+    public c: number = 0,
+  ) {
+    if (a + b <= c || c + b <= a || a + c <= b) {
```

Comment:
When throwing an error, provide a more detailed description in the error message. Instead of 'its not triangle', a better message might be 'Triangle inequality violated: the sum of the lengths of any two sides must be greater than the length of the remaining side.'